source/stylesheets/_landing.scss

```diff
@@ -222,7 +247,8 @@
   }
 
   .text {
-    margin-bottom: 3em;
+    margin: 0 auto 3em auto;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 auto 3em` instead of `0 auto 3em auto`